aks/application_configuration/resources.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  config_map_data = merge(
+    { SERVICE_TYPE = "web" },
```

Comment:
believe SERVICE_TYPE is apply specific, so would be passed to the module in the config_variables

Reply:
Ah right, I wondered if it was a Kubernetes thing, I'll remove it from here.